### PR TITLE
Update File with New Glossary Terms

### LINKED ISSUE #12: Replace old portal links with new doc portal ones
All previous eos.io developer portal links must be replaced with equivalent ENF doc portal links.

### PR DESCRIPTION
**Pull request by VEG-Ginna**
New glossary terms added to glossary

Resolves #12

### CODE REVIEW

**Review by iamveritas — CHANGES_REQUESTED** [review]
please replace all EOSIO with Antelope
see my comments below, I did not go through all occurrences for EOSIO.

File: glossary.md [hunk 1]
```diff
@@ -1,28 +1,35 @@
...
 
 ## Account Name
 An account name is a human-readable identifier that is stored on the blockchain. Standard account names can only contain the characters .abcdefghijklmnopqrstuvwxyz12345. a-z (lowercase), 1-5 and . (period), must start with a letter and must be 12 characters in length. Non-standard account names have all restrictions of the standard ones except they can have less than 12 characters in length.
 
 ## Account Name Bidding
 For non-standards accounts, those under 12 characters, EOSIO based blockchains provide a system where blockchain users can bid on accounts by submitting the bids via an auction process. Only one account name with the highest bidding price can be auctioned off every 24 hours among all submitted account names, that is, the system considers a bidding to be successful only when the bidding price is the highest among all other account bidding prices submitted.
```
[the remark this answers is not in the document]
Comment (VEG-Ginna) [reply glossary.md]: Done

File: glossary.md [hunk 2]
```diff
@@ -1,28 +1,35 @@
...
 
-## Airdrop (cryptocurrency airdrop)
-A marketing implementation that involves sending free coins or tokens to wallet addresses to promote awareness of a new currency.
-
-## Application Binary Interface
-A JSON-based description on how to convert user actions between their JSON and binary representations. The ABI may also describe how to convert the database state to/from JSON. Once you have described your contract via an ABI this allows developers and users to interact with your contract seamlessly via JSON.
-**Abbreviation**: ABI
-
 ## Action
 Functionality exposed by a smart contract that is exercised by passing the correct parameters via an approved transaction to an EOSIO network.
```
[the remark this answers is not in the document]
Comment (VEG-Ginna) [reply glossary.md]: Done

File: glossary.md [hunk 3]
```diff
@@ -1,28 +1,35 @@
...
+
+## Airgrab
+A method to deploy tokens for dApp developers. Use your personal RAM to receive airgrab tokens.
+
 ## Antelope
-The technology that powers EOS, WAX, Telos, UX, Ultra
-70% or EOSIO development funded by EOS
+The technology that powers EOS, WAX, Telos, UX, Ultra  
+70% or EOSIO development funded by EOS  
 30% of EOSIO development funded by Telos, WAX, and UX
```
[the remark this answers is not in the document]
Comment (VEG-Ginna) [reply glossary.md]: Done

**Review by iamveritas — APPROVED** [review]

**Pull request merged**